### DISCUSSION TITLE
Astro?

### DISCUSSION THREAD

**Discussion by WillEhrendreich** [amplifying-fsharp/amplifying-fsharp.github.io#39] [General]
Hey, I love the idea of promoting fsharp, thanks for spreading the good word about it, and encouraging people to get involved in contributing to open source! It's a fantastic idea. 

I'm hoping this is a good place to ask, I'm curious why when I open this up that I don't see any fsharp usage in it's implementation. 

I speak from a place of real, honest-to-God ignorance here, because I don't build web related ANYTHING, so please take what I'm saying with a grain of salt, and in the spirit of genuinely trying to be helpful because I believe in the idea of promoting fsharp. 

When I open up the page that promotes fsharp, and see that that page was built on Astro instead of something to do with the SAFE Stack or some fsharp based tech, I was pretty surprised.

I have no doubt whatsoever that Astro is probably wonderful. It seems like a great way to get a content based site up and going. It would be a fantastic choice for many people in many circumstances.

But opening up the github and seeing absolutely no fsharp was a lot like seeing a healthfood expert like 
![image](https://user-images.githubusercontent.com/55286472/234023271-bb5769ba-9ade-444a-8e68-a0f8040a9e6c.png)

And, you know, good for the ai generated imaginary healthfood lady, right? Like, she should let loose every now and again, she looks stressed, but, maybe she wouldn't want to have this pic on her site. lol.

**Comment by nojaf** [comment]
Hi Will,

Thanks for reaching out. I can understand how this looks 😅. So, yes, our website is all about contribution to F# and is ironically not made in F#.

The website is nothing more than some static content where I went with Astro out of curiosity.
I've been following the Astro team/story for some time now and I was curious about how well it works.
So, in short, I picked the hot front-end thing of the week to build our website.

The SAFE stack didn't come to mind as we don't do anything dynamically or need any back-end. The areas where F# shines are not being used. Other F# options would be [FSharp.Formatting](https://github.com/fsprojects/FSharp.Formatting) or perhaps even [Fornax](https://github.com/ionide/Fornax). `FSharp.Formatting` is more aimed at documentation and would not have been the best fit. And I've never used Fornax.

At the end of the day, I don't really think it matters that much. I can't see anything of our journey so far play out any differently if the static HTML and css of our website were created via some F# tool.

**Reply by WillEhrendreich** [reply]
sure, it probably doesn't mean much in the end, I just thought I'd throw it out there :) Thanks!
